packages/babel-parser/test/fixtures/typescript/export/declare-invalid/options.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sourceType": "module",
+  "plugins": [
+    "typescript"
+  ],
+  "throws": "foo (1:15)"
```

Comment:
We can throw a better error here. What about `"export declare" must be followed by an ambient declaration`?

You can add it to the `TSErrors` object, and then use `throw this.raise` rather than `this.unexpected`.

Reply:
Sure👍🏻